doc/tokens/authentication_modes.rst

```diff
@@ -1,15 +1,27 @@
 .. _authentication_modes:
+.. _client_modes:
 
-Authentication Modes
-====================
+Authentication Modes and Client Modes
+=====================================
 
 privacyIDEA supports a variety of tokens that implement different
 authentication flows. We call these flows *authentication modes*. Currently,
 tokens may implement three authentication modes, namely ``authenticate``,
 ``challenge`` and ``outofband``.
```

Comment:
No, we do not.
Maybe we should rename the class to AUTHENTICATIONMODE.

Otherwise we would have ``Authentication Mode``, which only occurs in the docs, ``TOkenmode``, which is the same as the authentication mode and the ``client mode`` which is more specific than the authentication mode and specificly tells the client how to react.

I will change the class name in the code, so that we only have ``authentication mode``, which describes the basic flow and the ``client mode`` which tells the client what to do.

Reply:
Will do

Reply:
Done in 9501114c (including typo in commit message)